Modules/MainFrame/MainFrame.lua

```diff
@@ -33,3 +34,15 @@ function Arenalogs.MAIN_FRAME:GetDisplayTeam()
 
     return mainFrame.content.matchHistoryTab.content.teamDisplayDropdown.selectedValue
 end
+]]
+--
+
+---@return character history based on selection FrameOptions? characterDropdown

```

Comment:
`---@return PlayerUID character <comment>`

where PlayerUID is the type (here its an alias for string)
and character is the name of the return variable